Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent(props) {
+    return (
+        <div class='card-content' >
+            <h2>Get Started with React</h2>
```

Comment:
So if you're passing the prop like I showed in the CardContainer file, you would then access the prop on this line 7 like `<h2>{prop.h2Prop}</h2>`.